### ISSUE TITLE
"Interface should be specified" when interface name is qualified

### ISSUE THREAD

**Issue by mthuurne** [Shoobx/mypy-zope#5]
When checking the following code:
```py
from twisted.internet import interfaces
from zope.interface import implementer

@implementer(interfaces.IPushProducer)
class MyProducer:
    pass
```
This error is reported:
```
error: Interface should be specified (should never happen)
```
It is possible to work around this by importing `IPushProducer` directly instead of importing its module.

**Comment by kedder** [comment]
Hm, do you have stubs for `twisted.internet`? The reason might be that mypy-zope does not recognize `IPushProducer` as an interface - it didn't see it declared anywhere.

**Comment by mthuurne** [comment]
I don't have stubs for Twisted. However, the problem goes away when rewriting the code like this:
```py
from twisted.internet.interfaces import IPushProducer
from zope.interface import implementer

@implementer(IPushProducer)
class MyProducer:
    pass
```
So while I cannot rule out that the lack of stubs is relevant, it's not the only factor here.

**Comment by kedder** [comment]
Interesting. I'll try to repro this in a test.
